File: src/main/java/com/procurement/submission/model/dto/response/BidsSelectionResponseDto.java
package com.procurement.submission.model.dto.response;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.procurement.submission.model.ocds.Bid;
import java.util.List;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class BidsSelectionResponseDto {

    private List<Bid> bids;

    @JsonCreator
    public BidsSelectionResponseDto(@JsonProperty("bids") final List<Bid> bids) {
        this.bids = bids;
    }
}

File: src/main/kotlin/com/procurement/submission/model/dto/response/BidsSelectionResponseDto.kt
package com.procurement.submission.model.dto.response

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude
import com.procurement.submission.model.dto.ocds.Bid

@JsonInclude(JsonInclude.Include.NON_NULL)
data class BidsSelectionResponseDto @JsonCreator constructor(

        val bids: Set<Bid>
)
